<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSRMDialog</class>
 <widget class="QDialog" name="OSRMDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OSRM: Get the fastest route with OSRM</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select the origin and the destination:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_about">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>About..</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="close_button_box">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>400</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>click on a category of point button and click on the map</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Provider:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_box_provider">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>5</y>
     <width>280</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOrigin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>88</y>
     <width>126</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Origin point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIntermediate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>128</y>
     <width>126</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Intermadiate point(s)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDest">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>169</y>
     <width>126</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Destination point</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_alternative">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>409</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Display possible alternatives roads</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReverse">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>409</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reverse O/D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClear">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>409</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear previous routes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTryIt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>409</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Display the result</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_instructions">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>409</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Display routing instructions</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_xyO">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>281</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_xyI">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>281</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_xyD">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>281</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <zorder>combo_box_provider</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_about</zorder>
  <zorder>close_button_box</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>pushButtonOrigin</zorder>
  <zorder>pushButtonIntermediate</zorder>
  <zorder>pushButtonDest</zorder>
  <zorder>checkBox_alternative</zorder>
  <zorder>pushButtonReverse</zorder>
  <zorder>pushButtonClear</zorder>
  <zorder>pushButtonTryIt</zorder>
  <zorder>checkBox_instructions</zorder>
  <zorder>lineEdit_xyO</zorder>
  <zorder>lineEdit_xyI</zorder>
  <zorder>lineEdit_xyD</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close_button_box</sender>
   <signal>accepted()</signal>
   <receiver>OSRMDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_button_box</sender>
   <signal>rejected()</signal>
   <receiver>OSRMDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
